packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -110,7 +144,34 @@ export const makeZoeHelpers = zcf => {
       zcf.complete(handles);
       return defaultAcceptanceMsg;
     },
-    // TODO update documentation to new API
+    /**
+     * Make an invitation to submit an Offer to this contract. This
+     * invitation can be given to a client, granting them the ability to
+     * participate in the contract.
+     *
+     * If the "expected" option is provided, it should be an ExpectedRecord.
+     * This is like a Proposal, but the amounts in 'want' and 'give' should be null,
+     * and the 'exit' should have a choice but the contents should be null.
+     * If the client submits an Offer which does not match these expectations,
+     * that offer will be rejected (and refunded).
+     *
+     * If "offerHook" is provided, it will be called when the invitation is exercised
+     * and an offer is submitted. The callback will get a reference to the offer.
+     *
+     * @param {InviteAnOfferOptions} [options={}]
+     * @returns {Invite}
+     *
+     * @typedef InviteAnOfferOptions
+     * @param {OfferHook} [offerHook]
+     * @param {CustomProperties} [customProperties]
+     * @param {ExpectedRecord} [expected]
+     *
+     * @typedef ExpectedRecord
+     * @property {TODO} [want]
```

Comment:
I think you still need to declare `TODO` in this file, like on line 39 of zoe.js
```js
* @typedef {any} TODO Needs to be typed
```

Reply:
added in b5f19200